test/data.js

```diff
@@ -225,7 +196,7 @@ const testData = Object.assign(
     sort: sortByUpdatedAtOrCreatedAtDesc,
     views: {
       simpleForms: (extendedForm) => {
-        const form = omit(extendedForm, ['lastSubmission', 'submissions']);
+        const form = R.omit(['lastSubmission', 'submissions'], extendedForm);
```

Comment:
be very very careful with rambda's omit ([explained here](https://github.com/opendatakit/central-backend/blob/master/lib/util/util.js#L27))